controller/src/main/java/io/pravega/controller/task/Stream/TxnSweeper.java

```diff
@@ -58,10 +60,16 @@ public TxnSweeper(final StreamMetadataStore streamMetadataStore,
         this.transactionMetadataTasks = transactionMetadataTasks;
         this.maxTxnTimeoutMillis = maxTxnTimeoutMillis;
         this.executor = executor;
+        this.ready = new AtomicBoolean(false);
+    }
+
+    public boolean isRunning() {
+        return ready.get();
```

Comment:
Could we rename this methods to `isReady()` as is the case with other sweepers, because `isRunning` is typically defined on a service but our sweepers have utility sweep methods that are invoked from the ClusterListener service.
Moreover, instead of adding a new `ready` variable, we could reuse the one in transactionMetadataTasks and define this method as `transactionMetadataTasks.isReady()`, similar to how we have defined `awaitInitialization` method as `transactionMetadataTasks.awaitInitialization`.